tools/accuracy_checker/accuracy_checker/annotation_converters/coco_facial_landmarks.py

```diff
@@ -0,0 +1,86 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .format_converter import FileBasedAnnotationConverter, ConverterReturn
+from ..representation import Face98LandmarksAnnotation
+from ..utils import read_xml, check_file_existence
+from ..config import PathField
+import json
+
+
+class COCOFacialLandmarksRecognitionConverter(FileBasedAnnotationConverter):
+    __provider__ = 'coco_facial_landmarks'
+    annotation_types = (Face98LandmarksAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'images_dir': PathField(
+                is_directory=True, optional=True,
+                description='path to dataset images, used only for content existence check'
+            )
+        })
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.images_dir = self.get_value_from_config('images_dir') or self.annotation_file.parent
+
+    def _collectImageIds(self, data):
+        result = {}
+        ids = []
+        for itm in data:
+            img_name = itm["file_name"]
+            id = itm["id"]
+            ids.append(id)
+            result[id] = img_name
+
+        return result, ids
+
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        
+        with open(self.annotation_file) as f:
+            data = json.load(f)
+        coco_ann = data["annotations"]
+        id2name, img_ids = self._collectImageIds (data["images"])
+        num_images = len(img_ids)
+        num_landmarks = 98
+        annotations = []
+        for ann in coco_ann[1:]:
```

Comment:
why is it started from 1?

Reply:
Annotation feature... There is image id without landmarks. xtcocotools resolves this problem. But I need to do it manually.